File: src/main/java/dev/fynf/randomfood2/entities/ModifierRepository.java
package dev.fynf.randomfood2.entities;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface ModifierRepository extends CrudRepository<Modifier, Long> {
  List<Modifier> findAllByIdNotIn(List<Long> ids);

}

File: src/main/kotlin/dev/fynf/randomfood2/entities/ModifierRepository.kt
package dev.fynf.randomfood2.entities

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface ModifierRepository : CrudRepository<Modifier?, Long?> {
    fun findAllByIdNotIn(ids: List<Long?>?): List<Modifier?>?
}